src/windows/functions.cpp

```diff
@@ -393,6 +393,12 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveLetterValid(TCHAR driveLetter) {
+  int driveLetterIndex = static_cast<int>(driveLetter) - static_cast<int>('A');
```

Comment:
> This triggers the same problematic error despite the fix

Looking at your latest changes, perhaps this problem was also triggered by a user having a CDROM as D: and their removable flash drive as E:, which I guess is likely to be a common scenario.